components/style/themes/default.less

```diff
@@ -290,6 +290,10 @@
 @table-padding-vertical: 16px;
 @table-padding-horizontal: 8px;
 
+// Tag
+// --
+@tag-default-bg: @background-color-base;
```

Comment:
`#f3f3f3` and `#f7f7f7` are almost identical colors and this will make the tags a bit more consistent

![image](https://cloud.githubusercontent.com/assets/3475472/25706777/6c22c934-30af-11e7-9f16-b66ab4332149.png)
![image](https://cloud.githubusercontent.com/assets/3475472/25706797/74be504a-30af-11e7-9be0-2c4002bfcd31.png)